README.md

```diff
@@ -1,79 +1,66 @@
 # Microsoft Teams JavaScript client SDK
 
-Welcome to the Teams client SDK monorepo! For breaking changes, please refer to our changelog in the monorepo root of the `2.0-preview` branch.
-
-This JavaScript library is part of the [Microsoft Teams developer platform](https://developer.microsoft.com/microsoft-teams/). See full [SDK reference documentation](https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client).
-
-[![Build Status](https://travis-ci.org/OfficeDev/microsoft-teams-library-js.svg?branch=master)](https://travis-ci.org/OfficeDev/microsoft-teams-library-js)
-[![Coverage Status](https://coveralls.io/repos/github/OfficeDev/microsoft-teams-library-js/badge.svg?branch=master)](https://coveralls.io/github/OfficeDev/microsoft-teams-library-js?branch=master)
+Welcome to the Teams client SDK monorepo! For breaking changes, please refer to our changelog in the monorepo root of the `2.0-preview` branch. This repository contains the core teams-client-sdk as well as tools and applications for analyzing and testing.
```

Comment:
I don't think we ever use "teams-client-sdk" anywhere. Can you please refer to it at "teams-js", since that is its name defined in the package.json? Or "@microsoft/teams-js" works too.

Reply:
of course!

Reply:
i was just using the names that were in the repo so that external people would understand.

Reply:
this was already there so I didn't think to change it but this makes a lot more sense since this is a whole rewrite

Reply:
i saw it listed as the client sdk here also: https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client?view=msteams-client-js-beta#finding-the-sdk FWIW we should really just stick to one name to be less confusing (also this group is called teams-client-sdk-owners too 😆 )